client/src/components/AboutUkraine/AboutUkraine.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react'
+import {Box, Typography, styled, keyframes  } from '@mui/material';
+// import { useInView } from 'react-intersection-observer';
+
+const UkraineTextInfo = styled((props) => <Typography variant="body1" {...props} />)(({ theme }) => ({
+    [theme.breakpoints.up('tablet')]: {
+        padding: '10px',
+        fontSize: '18px',
+        lineHeight: '1.2',
+    },
+
+    [theme.breakpoints.up('laptop')]: {
+        fontSize: '18px',
+        padding: '20px'
+    },
+}));
+
+const AboutUkraineContainer = styled((props) => <Box {...props} />)(({ theme }) => ({
+    backgroundColor: '#EDEDED',
+    borderRadius: '20px',
+    zIndex: 1,
+    transition: '2s',
+    position: 'absolute',
+    top: '12%',
+    left: '40%',
+    width: '200px',
+    marginRight: '30px',
+    padding: '20px',
+   
+    '&:hover':{
+        backgroundColor: '#0499dd',
+        animation: `
+        -webkit-transform: translateX(-60px);
+        transform: translateX(-60px);`,
+        boxShadow: '1px 1px #ededed, 2px 2px #ededed, 3px 3px #ededed, 4px 4px #ededed, 5px 5px #ededed, 6px 6px #ededed,  7px 7px #ededed',
+    },
+
+    '&:hover h2':{
+       color:'white'
```

Comment:
our secondary color for hover is yellow.

```suggestion
       color: theme.palette.secondary.main
```

Reply:
done